android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/social/wechat/WechatSocialResultActivity.kt

```diff
@@ -0,0 +1,55 @@
+package io.logto.sdk.android.auth.social.wechat
+
+import android.app.Activity
+import android.net.Uri
+import android.os.Bundle
+import com.tencent.mm.opensdk.modelbase.BaseReq
+import com.tencent.mm.opensdk.modelbase.BaseResp
+import com.tencent.mm.opensdk.openapi.IWXAPIEventHandler
+import com.tencent.mm.opensdk.openapi.WXAPIFactory
+
+open class WechatSocialResultActivity : Activity(), IWXAPIEventHandler {
+    companion object {
+        private const val WECHAT_SOCIAL_SESSION_KEY = "wechat"
+        private val wechatSocialSession = mutableMapOf<String, WechatSocialSession>()
```

Comment:
There seems to be only one key-value pair in this map.
Why not just use a `wechatSocialSession` for now?